core/blockchain.go

```diff
@@ -1535,30 +1531,6 @@ func (bc *BlockChain) InsertChain(chain types.Blocks) (int, error) {
 	return bc.insertChain(chain, true)
 }
 
-func findVerkleConversionBlock() (uint64, error) {
```

Comment:
This was removed since we don't rely on the old `.txt` anymore but in timestamps.